user_tools/src/spark_rapids_tools/tools/cluster_config_recommender.py

```diff
@@ -221,17 +213,3 @@ def fill_na_fn(x, def_val):
             self.logger.warning('Error while adding cluster and tuning recommendations. Reason - %s:%s',
                                 type(e).__name__, e)
             return tools_processed_apps
-
```

Comment:
We do not need to print the cluster conversion string since will display it in the TCV table